### ISSUE TITLE
Custom scrollbars show on full screen YouTube videos

### ISSUE THREAD

**Issue by Susko3** [spencerwooo/firefox-overlay-scrollbar#6]
When I full screen a YouTube video, the scrollbar is visible on the screen, which is very distracting.

Would it be possible to make a website blacklist so that the custom scrollbars are disabled on those websites. Or, even better, a way to make the scrollbars invisible on certain websites.

**Comment by jenabaivab** [comment]
Yes, facing this issue as well. Please, is there a way to make sure it doesn't appear in full screen mode?

**Comment by Susko3** [comment]
@jenabaivab 
Found a easy workaround for this: enter fullscreen using firefox's video fullscreen, instead of youtube's.

Right click twice on the video, and then select `Full Screen` (you can press <kbd>F</kbd> to quickly select the fullscreen option)

**Comment by brian6932** [comment]
Bump, this is really annoying

**Comment by jenabaivab** [comment]
Any potential fixes for this?

**Comment by spencerwooo** [comment]
Sorry, not using firefox or windows for the time being ;( Actually, I'm going to archive this repo as of 2022.
